crates/nu-command/src/filters/first.rs

```diff
@@ -145,20 +148,11 @@ fn first_helper(
                         .set_metadata(metadata)),
                 }
             }
-            _ => {
-                if rows_desired == 1 {
-                    match input_peek.next() {
-                        Some(val) => Ok(val.into_pipeline_data()),
-                        None => Err(ShellError::AccessBeyondEndOfStream(head)),
-                    }
-                } else {
-                    Ok(input_peek
-                        .into_iter()
-                        .take(rows_desired)
-                        .into_pipeline_data(ctrlc)
-                        .set_metadata(metadata))
-                }
-            }
+            _ => Ok(input_peek
+                .into_iter()
+                .take(rows_desired)
+                .into_pipeline_data(ctrlc)
+                .set_metadata(metadata)),
```

Comment:
Based on what the change appears to be doing, this will once again break consistency with `last`: `first 1` will now return a list, just as `last 1` already did, but `first` (without a positional param) will now also return a list whereas `last` (without a positional param) will return an element.

Compare to https://github.com/nushell/nushell/blob/483f007b092fee3cfcdc77461daba9f838fd6594/crates/nu-command/src/filters/last.rs#L34-L46

The only real difference is that `last` (prior to this PR) had the more self-consistent implementation.